riotfile.py

```diff
@@ -654,6 +666,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "python-memcached": latest,
                 "redis": "~=2.0",
                 "blinker": latest,
+                "werkzeug": "<2.0",
```

Comment:
```suggestion
                "werkzeug": "<1.0",
```

Flask-Cache relies on `werkzeug.import_string` which was removed in 1.0

Reply:
ah, good catch!